src/core/server/plugins/plugin_context.ts

```diff
@@ -37,6 +37,10 @@ import {
 import { pick, deepFreeze } from '../../utils';
 import { CoreSetup, CoreStart } from '..';
 
+export interface InstanceInfo {
+  uuid: string;
+}
```

Comment:
Added a structure as the `pid` will also be added here once we migrate it from the legacy server